README.md

```diff
@@ -106,58 +143,63 @@ datadog.addForwarderToNonLambdaLogGroups([<LOG_GROUPS>])
   - `datadog-cdk-constructs` >= 0.8.5
 
 ### Alternative Methods to Enable Source Code Integration
-If the automatic implementation doesn't work for your case, please follow one of the two guides below. 
+
+If the automatic implementation doesn't work for your case, please follow one of the two guides below.
 
 **Note: these alternate guides only work for Typescript.**
+
 <details>
   <summary>datadog-cdk version satisfied, but Datadog Github Integration NOT installed</summary>
 
-  If the Datadog Github Integration is not installed, you need to import the `datadog-ci` package and manually upload your Git metadata to Datadog.
-  We recommend you do this where your CDK Stack is initialized.
+If the Datadog Github Integration is not installed, you need to import the `datadog-ci` package and manually upload your Git metadata to Datadog.
+We recommend you do this where your CDK Stack is initialized.
```

Comment:
```suggestion
For the best results, import the `datadog-ci` package where your CDK Stack is initialized.
```
Feel free to replace "For the best results" with a description of the benefits of having the package and CDK Stack in the same place.